Monika After Story/game/script-topics.rpy

```diff
@@ -9831,72 +9834,30 @@ label monika_vehicle:
-                        m 3eub "A car like that would be great for driving around the city, don't you think, [player]?"
-                        m 3eua "We could go to museums, parks, malls and so on."
-                        m 1eua "It'd be so nice to be able to drive to places that are too far to walk to by foot."
-                        m 3hua "It's always exhilarating to discover and explore new places."
-                        m 1rksdla "We might even find a place where the both of us can be together..."
-                        m 1tsu "...Alone."
-                        m 1hub "Ahaha!"
-                        m 3eua "Just so you know, I'm expecting more than just a simple drive around the city for our dates..."
-                        m 1hua "I hope you'll surprise me, [player]."
-                        m 1hub "But then again...{w=0.5}I'd love anything as long as it's with you~"
-
-                    "Motorcycle.":
-                        $ persistent._mas_pm_owns_car_type = "motorcyle"
-
-                        m 1hksdlb "Eh?"
-                        m 1lksdlb "You drive a motorcycle?"
-                        m 1eksdla "I'm surprised, I never expected that to be your kind of ride."
-                        m 1lksdlb "To be honest, I'm a little hesitant to ride one, ahaha!"
-                        m 1eua "Really, I shouldn't be scared..."
-                        m 3eua "You're the one driving after all."
-                        m 1lksdla "That puts my mind at ease...{w=1}a little."
-                        m 1eua "Just take it nice and slow, okay?"
-                        m 3hua "After all, we aren't in any rush."
-                        m 1tsu "Or...{w=1}was it your plan to drive fast, so that I would hang on to you tightly?"
-                        m 3tsu "That's pretty sneaky of you, [player]."
-                        m 1hub "Ehehe~"
-                        m 3eka "There's no need to be shy, my love."
-                        m 3ekbsa "I'll hug you, even if you don't ask for it..."
-                        m 1hkbfa "That's how much I love you~"
+                python:
+                    option_list = [
+                        ("An SUV.", "monika_vehicle_suv",False,False),
+                        ("A pickup truck.","monika_vehicle_pickup",False,False), #note, doing this to give the illusion of two options
+                        ("A sports car.","monika_vehicle_sportscar",False,False),
+                        ("A sedan.","monika_vehicle_sedan",False,False),
+                        ("A hatchback.","monika_vehicle_hatchback",False,False),
+                        ("A motorcycle.","monika_vehicle_motorcycle",False,False),
+                        ("Another vehicle.","monika_vehicle_other",False,False)
+                    ]
+
+                #Display our scrollable
+                show monika at t21
+
+                m "Is it any of the vehicles I mentioned, or is it something else?{fast}"
```

Comment:
the dual question thing is only for regular menus. for programmatic menus like this one, just do the question once without any extra text tags.